dlt/extract/incremental/__init__.py

```diff
@@ -166,6 +178,17 @@ def __init__(
         self._bound_pipe: SupportsPipe = None
         """Bound pipe"""
 
+    def to_table_hint(self) -> Optional[IncrementalArgs]:
+        """Table hint is only returned when all properties are serializable"""
+        if self.last_value_func not in (min, max):
```

Comment:
we we do not need to care. we just return True if we can localize incremental to a column